README.md

```diff
@@ -1,19 +1,27 @@
-# The manuscript bot for automated scholarly publishing
+# Python utilities for Manubot: Manuscripts, open & automated
 
 [![Travis Linux Build Status](https://travis-ci.com/manubot/manubot.svg?branch=master)](https://travis-ci.com/manubot/manubot)
 [![AppVeyor Windows Build Status](https://ci.appveyor.com/api/projects/status/f20hvc6si5uiqd7e/branch/master?svg=true)](https://ci.appveyor.com/project/manubot/manubot/branch/master)
 
-The Manubot Python package prepares scholarly manuscripts for Pandoc consumption.
-It automates and scripts several aspects of manuscript creation, including fetching bibliographic metadata for citations.
+[Manubot](https://manubot.org/ "Manubot homepage") is a workflow and set of tools for the next generation of scholarly publishing.
+This repository contains a Python package that contains several Manubot-related utilities.
 
-This program is designed to be used with clones of [Manubot Rootstock](https://github.com/greenelab/manubot-rootstock), which perform Pandoc conversion and continuous deployment.
-See the Manubot Rootstock [usage guide](https://github.com/greenelab/manubot-rootstock/blob/master/USAGE.md) for more information.
+Specifically, the Python package includes a `manubot process` command line utility to prepares scholarly manuscripts for Pandoc consumption.
+This utility automates several aspects of manuscript creation, including fetching bibliographic metadata for citations, and is used to build Manubot manuscripts.
+See the [manuscript usage guide](https://github.com/manubot/rootstock/blob/master/USAGE.md) for more information.
+
```

Comment:
Do you also want to note the command line utility here to indicate there is more functionality besides `manubot process`?